src/highdicom/sr/enum.py

```diff
@@ -4,77 +4,199 @@
+
     POLYLINE = 'POLYLINE'
+    """A series of connected line segments with ordered vertices denoted by
+    (x,y,z) triplets; the points need not be coplanar."""
+
     POLYGON = 'POLYGON'
+    """A series of connected line segments with ordered vertices denoted by
+    (x,y,z) triplets, where the first and last vertices shall be the same
+    forming a polygon; the points shall be coplanar."""
 
 
 class TemporalRangeTypeValues(Enum):
 
-    """Enumerated values for attribute Temporal Range Type."""
+    """Enumerated values for attribute Temporal Range Type.
+
+    See :dcm:`C.18.7.1.1 <part03/sect_C.18.7.html#sect_C.18.7.1.1>`.
+
+    """
 
     BEGIN = 'BEGIN'
+    """A range beginning at one temporal point, and extending beyond the end of
+    the acquired data."""
+
     END = 'END'
+    """A range beginning before the start of the acquired data, and extending
+    to (and including) the identified temporal point."""
+
     MULTIPOINT = 'MULTIPOINT'
+    """Multiple temporal points."""
+
     MULTISEGMENT = 'MULTISEGMENT'
+    """Multiple segments, each denoted by two temporal points."""
+
     POINT = 'POINT'
+    """A single temporal point."""
+
     SEGMENT = 'SEGMENT'
+    """A range between two temporal points."""
 
 
 class RelationshipTypeValues(Enum):
 
-    """Enumerated values for attribute Relationship Type."""
+    """Enumerated values for attribute Relationship Type.
+
+    See :dcm:`C.17.3.2.4 <part03/sect_C.17.3.2.4.html#sect_C.17.3.2.4>`.
+
+    """
 
     CONTAINS = 'CONTAINS'
+    """Source Item contains Target Content Item."""
+
     HAS_ACQ_CONTEXT = 'HAS ACQ CONTEXT'
+    """The Target Content Item describes the conditions present during data
+    acquisition of the Source Content Item."""
+
     HAS_CONCEPT_MOD = 'HAS CONCEPT MOD'
+    """Used to qualify or describe the Concept Name of the Source Content Item,
+    such as to create a post-coordinated description of a concept, or to
+    further describe a concept."""
+
     HAS_OBS_CONTEXT = 'HAS OBS CONTEXT'
+    """Target Content Items shall convey any specialization of Observation
+    Context needed for unambiguous documentation of the Source Content Item."""
+
     HAS_PROPERTIES = 'HAS PROPERTIES'
+    """Description of properties of the Source Content Item."""
+
     INFERRED_FROM = 'INFERRED FROM'
+    """Source Content Item conveys a measurement or other inference made from
+    the Target Content Items. Denotes the supporting evidence for a measurement
+    or judgment."""
+
     SELECTED_FROM = 'SELECTED FROM'
+    """Source Content Item conveys spatial or temporal coordinates selected
+    from the Target Content Item(s)."""
 
 
 class PixelOriginInterpretationValues(Enum):
 
     """Enumerated values for attribute Pixel Origin Interpretation."""
 
     FRAME = 'FRAME'
+    """Relative to the individual frame."""
+
     VOLUME = 'VOLUME'
+    """Relative to the Total Image Matrix (for tiled images)."""
```

Comment:
The term `"VOLUME"` is very confusing in this context. Attempt to clarify:

```suggestion
    """Relative to the Total Pixel Matrix of the VOLUME image."""
```